api/envoy/extensions/filters/http/oauth2/v3/oauth.proto

```diff
@@ -148,7 +148,7 @@ message OAuth2Config {
 
   // If set to true, allows automatic access token refresh using the associated refresh token (see
   // `RFC 6749 section 6 <https://datatracker.ietf.org/doc/html/rfc6749#section-6>`_), provided that the OAuth server supports that.
-  // Default value is false.
+  // Default value is true.
```

Comment:
It's ok to change the default value to a more appropriate value. But please guard this by a runtime flag.

Reply:
Ack, I didn't think this is a high impact change. Added now

Reply:
Yeah, it's not big impact, although it's should not a very minor change. So, be careful would be better anyway.

Reply:
Agreed! Thanks a lot for taking a look at this :D